Common/include/option_structure.hpp

```diff
@@ -371,13 +371,17 @@ enum ENUM_KIND_NONDIM {
   DIMENSIONAL = 0,			    /*!< \brief Dimensional simulation. */
   FREESTREAM_PRESS_EQ_ONE = 1, /*!< \brief Non-dimensional simulation. */
```

Comment:
Could you please update these comments... I'm worried about the fact that SU2 produce different results depending on the non-dimensionalization. And I think is because in the Venkat limiter we are not careful with the epsilon. Have you ever looked at that? I think that DIMENSIONAL is the default for compressible. Are you using the same for incompressible?